jetty-documentation/src/main/asciidoc/administration/sessions/chapter.adoc

```diff
@@ -17,9 +17,21 @@
 [[session-management]]
```

Comment:
Lets also remember to mention somewhere the command line option --skip-file-validation when starting. Useful when you've updated the versions of the jars that jetty has automatically downloaded for you.